src_c/base.c

```diff
@@ -2295,6 +2302,11 @@ MODINIT_DEFINE(base)
 #endif
 #endif
     }
+
+    if (!pg_CheckSDLVersions()) {
```

Comment:
Can this check come somewhere top in the module init code? That would help to error faster

Reply:
Possibly. I did try putting it as early as possible in the init code and hit some error that I don't recall (I ran it in a container in a pretty ad-hoc way, don't have the output, sorry :disappointed:)

Reply:
Possibly. I did try putting it as early as possible in the init code and hit some error that I don't recall (I ran it in a container in a pretty ad-hoc way, don't have the output, sorry :disappointed:)

Reply:
hmmm... I think this check would be useless if it is called late. Atleast, it should be called before any other SDL API is called. In this case, `SDL_RegisterApp` is called before this function on windows, so this check should happen before that.

Additionally, `pg_CheckSDLVersions` is also called in the `pg_init` function, which will be redundant and should be removed, if this check already happens in the module init function

Reply:
It's still useful even if late, because there could be many bugs that happen in actual usage of pygame after all the init. Merging in this state (and making another issue to address later) would still be an improvement on the status quo.